packages/polaris-viz/src/components/LineChart/stories/playground/Playground.stories.tsx

```diff
@@ -15,6 +177,45 @@ const Template: Story<LineChartProps> = (args: LineChartProps) => {
   return <LineChart {...args} />;
 };
 
+const getName = (reportifyValue) => {
+  const nameTest = Date.parse(reportifyValue);
+  if (isNaN(nameTest)) {
+    return reportifyValue;
+  } else {
+    const parsedDate = new Date(reportifyValue);
+    return `${parsedDate.getFullYear()}-${String(
+      parsedDate.getMonth() + 1,
+    ).padStart(2, '0')}-${String(parsedDate.getDate()).padStart(2, '0')}`;
+  }
+};
+
+const parseReportify = (reportifyDataObj) => {
+  const parsedData: DataSeries[] = [];
+  const oganizedData: DataSeries[] = [];
+  const reportifyData = reportifyDataObj.result.data;
+  for (let index = 0; index < reportifyData.length; index++) {
+    const [date, key, percentValue] = reportifyData[index];
+    const formattedValue = parseFloat((percentValue * 100).toFixed(3));
+    if (date in oganizedData) {
+      oganizedData[date].data.push({key: key, value: formattedValue});
+    } else {
+      const cleanName = getName(date);
+      oganizedData[date] = {
+        data: [{key: key, value: formattedValue}],
+        name: cleanName,
+      };
+    }
+  }
+  for (const property in oganizedData) {
+    parsedData.push(oganizedData[property]);
+  }
+  return parsedData;
+};
+
+const COHORT_DATA = parseReportify(REPORTIFY_RESPONSE_AVG[0]).concat(
+  parseReportify(REPORTIFY_RESPONSE_COHORT[0]),
+);
+
```

Comment:
I wonder if we could remove all this parsing and just pass the hardcoded data already formatted.